src/environment/stella_environment.cpp

```diff
@@ -178,6 +178,12 @@ void StellaEnvironment::soft_reset(){
   m_state.incrementFrame();
 }
 
+/** Emulates a NOOP action for both players */
+void StellaEnvironment::wait()
```

Comment:
For sure, it shouldn't. But what I am proposing is to move the `wait` function to BreakoutSuper.cpp. It shouldn't change the behavior of anything else and the code given in doc/example/.. should work just fine. Does it make sense?